src/sentry/static/sentry/app/components/sidebar/sidebarItem.jsx

```diff
@@ -9,6 +9,13 @@ import Link from '../link';
 import TextOverflow from '../textOverflow';
 import Tooltip from '../tooltip';
 
+const LabelHook = HookOrDefault({
```

Comment:
is it safe to call hooks at the top level? how do we know the hooks will be loaded at this point?

Reply:
This actually doesn't call a hook, it just composes a component that will call the hook on render.

Reply:
👍